perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -518,6 +592,178 @@ def SingleStreamThroughputBenchmark(storage_schema, host_to_connect=None):
                   host_to_connect=host_to_connect)
 
 
+def RunMultiThreadedWorkersWriteResults(worker, worker_args,
```

Comment:
minor: how about just "RunThreadedWorkers"? (threaded implies multi-threaded, "write results" is an implementation detail and kind of implied too: we will get results out of this function)